subworkflows/local/post_variantcalling/main.nf

```diff
@@ -2,26 +2,38 @@
 // POST VARIANT CALLING: processes run on variantcalled but not annotated VCFs
 //
 
-include { CONCATENATE_GERMLINE_VCFS } from '../vcf_concatenate_germline/main'
+include { CONCATENATE_GERMLINE_VCFS } from '../vcf_concatenate_germline'
+include { NORMALIZE_VCFS            } from '../vcf_normalization'
 
 workflow POST_VARIANTCALLING {
-
     take:
-    vcfs
+    germline_vcfs
+    tumor_only_vcfs
+    somatic_vcfs
+    fasta
     concatenate_vcfs
+    normalize_vcfs
 
     main:
     versions = Channel.empty()
+    vcfs = Channel.empty()
 
-    if (concatenate_vcfs){
-        CONCATENATE_GERMLINE_VCFS(vcfs)
+    if (concatenate_vcfs) {
+        CONCATENATE_GERMLINE_VCFS(germline_vcfs)
 
         vcfs = vcfs.mix(CONCATENATE_GERMLINE_VCFS.out.vcfs)
         versions = versions.mix(CONCATENATE_GERMLINE_VCFS.out.versions)
     }
 
-    emit:
-    vcfs // post processed vcfs
+    if (normalize_vcfs) {
+        NORMALIZE_VCFS(germline_vcfs, tumor_only_vcfs, somatic_vcfs, fasta)
+
+        vcfs = vcfs.mix(NORMALIZE_VCFS.out.vcfs)
 
+        versions = versions.mix(NORMALIZE_VCFS.out.versions)
+    }
```

Comment:
So given this, I'd say if we concatenate_vcfs and normalize_vcfs in one go it's similar as if we were concatenate_vcfs and normalize_vcfs separately.
We don't normalize the concatenated vcfs

Reply:
ok so the docs should say something about that I think. the metromap looks like you can optionally normalise the concatenated vcfs